digital-inverter-openlane.ipynb

```diff
@@ -223,10 +194,10 @@
             "env: OPENLANE_ROOT=/content/OpenLane\n",
             "env: PATH=/content/conda-env/bin:/opt/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/tools/node/bin:/tools/google-cloud-sdk/bin:/content/OpenLane:/content/OpenLane/scripts\n",
             "env: OPENLANE_LOCAL_INSTALL=1\n",
-            "OpenLane 01e951092150ee8619286b0807ee263198b5ea6d\r\n",
```

Comment:
as the switch to minimamba was the most important change, maybe we could drop changes to output? Or do you explicitly want to store outputs on the repo?

Reply:
I think it's interesting to have the output in the ipynb version so that you can preview it in the github ui https://github.com/chipsalliance/silicon-notebooks/blob/main/digital-inverter-openlane.ipynb and in https://developers.google.com/silicon/guides/digital-inverter-openlane

But ultimatly I'd like get rid of it in the source file when we go thru https://github.com/chipsalliance/silicon-notebooks/issues/3